com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -196,6 +196,7 @@ public bool IsCameraProjectionMatrixFlipped()
     [MovedFrom("UnityEngine.Rendering.LWRP")] public struct ShadowData
     {
         public bool supportsMainLightShadows;
+        [Obsolete("Obsolete, this feature will be supported by 'RendererFeature' in the future")]
```

Comment:
Update text to mention new ScreenSpaceShadow renderer feature.

Reply:
Applied it :)